### ISSUE TITLE
Lumen config_path() error, commit #1d1aef1

### ISSUE THREAD

**Issue by Donatello-za** [barryvdh/laravel-cors#186]
This commit https://github.com/barryvdh/laravel-cors/commit/1d1aef1bb220709817307223a3d048c87e872636 introduced an error in Lumen 5.4 as the config_path() helper function is not available in Lumen (Thanks Laravel Devs! :P)

You could perhaps do the following on line 35 of ServiceProvider.php:

```php
$this->publishes([$this->configPath() => $this->app->getConfigurationPath('cors.php')]);
```
instead of:
```php
$this->publishes([$this->configPath() => config_path('cors.php')]);
```

**Comment by barryvdh** [comment]
Should already be fixed with latest version (0.9.3)

**Comment by Donatello-za** [comment]
Thanks, I tested it and it seems fine now, I will close the issue.
